testing/python/fixture.py

```diff
@@ -490,6 +490,21 @@ def test_func(something): pass
         print(ss.stack)
         assert teardownlist == [1]
 
+    def test_mark_as_fixture_with_prefix_and_decorator_fails(self, testdir):
+        testdir.makeconftest("""
+            import pytest
+
+            @pytest.fixture
+            def pytest_funcarg__marked_with_prefix_and_decorator():
+                pass
+        """)
+        result = testdir.runpytest_subprocess()
+        assert result.ret != 0
+        result.stdout.fnmatch_lines([
+            "*AssertionError: *",
+            "*pytest_funcarg__marked_with_prefix_and_decorator*"
```

Comment:
Checking for both occurrences works as expected. Also just checking for each single occurrence. If I change the check to :

>         result.stdout.fnmatch_lines([
>           "*AssertionError*pytest_funcarg__marked_with_prefix_and_decorator*"
>         ])
>     E       Failed: remains unmatched: 
>     '*AssertionError*pytest_funcarg__marked_with_prefix_and_decorator*', see stderr

So I guess it's o.k. to explicitly check first for the assertion error and then for the wanted function name?